extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/OpenConnections.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.websockets.next;
+
+import java.util.Collection;
+import java.util.stream.Stream;
+
+import io.smallrye.common.annotation.Experimental;
+
+/**
+ * Provides convenient access to all open connections from clients to {@link WebSocket} endpoints on the server.
+ *
+ * @see WebSocketConnection
+ */
+@Experimental("This API is experimental and may change in the future")
+public interface OpenConnections {
```

Comment:
Alternatively, we could provide a bean of type `Set<WebSocketConnection>`. This would have to be a stateless delegating proxy though, because the underlying set of open connections is dynamic. And injecting an immutable snapshot is impractical for beans with wider scope, e.g. `@ApplicationScoped`.

The other downside is the impossibility to define the contract in the javadoc.

Reply:
I think `OpenConnections` makes perfect sense